leaflet/templates/leaflet/admin/change_list_geo.html

```diff
@@ -0,0 +1,99 @@
+{% extends 'admin/change_list.html' %}
+{% load leaflet_tags %}
+{% load admin_list_leaflet %}
```

Comment:
I just doing the test,
- I with an error : 'admin_list_leaflet' is not a valid tag library: Template library admin_list_leaflet not found, tried
- Without no menus and no layer
![nolistmapadmin](https://cloud.githubusercontent.com/assets/16896831/12607783/e3f87c78-c4d7-11e5-8d3e-213455c04836.png)